rts/Lua/LuaSyncedCtrl.h

```diff
@@ -174,5 +176,21 @@ class LuaSyncedCtrl
 		static int SetRadarErrorParams(lua_State* L);
 };
 
+//Stores Pieces Hierarchy Information - HigherPiece is the Parent, SubPieces the direct Children
+struct PieceHierarchy
+{
+	float HigherPiece;
+	std::vector<float> * SubPieces;
```

Comment:
i didn't really investigate the code, but at a quick look:

1. why float? aren't piece numbers int/unsigned int/size_t?
2. why a pointer?

should do the same:
std::vector<size_t> SubPieces;